src/app/pages/office/office.model.ts

```diff
@@ -0,0 +1,5 @@
+export type OfficeModel = {
```

Comment:
If the ```officeReducer``` is removed then this file can also be deleted - see below re deleting ```officeReducer```

Reply:
agree, removed